input_files/simple_dsp_slice_65nm/dsp_slice.txt

```diff
@@ -0,0 +1,151 @@
+#This is a sample settings file for the hard block flow
```

Comment:
Adding a reference to the COFFE 2 paper for more info would probably be useful.

Ref is here: https://dl.acm.org/doi/10.1145/3301298